### PR TITLE
header component - added admin - page name to header

### PR DESCRIPTION
**Pull request by naveenbhaskar**

### CODE REVIEW

**Comment by naveenbhaskar** [comment]
image

File: .idea/codeStyles/codeStyleConfig.xml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <component name="ProjectCodeStyleConfiguration">
   <state>
     <option name="USE_PER_PROJECT_SETTINGS" value="true" />
+    <option name="PREFERRED_PROJECT_CODE_STYLE" value="Default (1)" />
```
[the remark this answers is not in the document]
Comment (naveenbhaskar) [reply .idea/codeStyles/codeStyleConfig.xml]: oops.. will remove this and update

File: server/webapp/WEB-INF/rails/webpack/views/pages/new_plugins/index.scss.d.ts [hunk 2]
```diff
@@ -2,6 +2,7 @@
 // Please do not change this file!
 export const pluginHeader: string;
 export const pluginIcon: string;
+export const pluginName: string;
```
Comment (GaneshSPatil) [inline server/webapp/WEB-INF/rails/webpack/views/pages/new_plugins/index.scss.d.ts]: Not required. Already done as part of https://github.com/gocd/gocd/commit/afa4c88794477caccc98c5765cbfb54c015edf27 commit

File: server/webapp/WEB-INF/rails/webpack/views/components/header_panel/index.tsx [hunk 3]
```diff
@@ -22,6 +22,7 @@ import * as style from "./index.scss";
 
 export interface Attrs {
   title: m.Children;
+  sectionName: m.Children;
```
Comment (GaneshSPatil) [inline server/webapp/WEB-INF/rails/webpack/views/components/header_panel/index.tsx]: can we make `sectionName` optional? as all the pages will not have a `sectionName`

**Comment by akshaydewan** [comment]
Made sectionName optional and reverted `codeStyleConfig.xml`

**Pull request merged**